integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/TestQueryWithColumnMetCacheAndCacheLevelProperty.scala

```diff
@@ -340,5 +340,15 @@ class TestQueryWithColumnMetCacheAndCacheLevelProperty extends QueryTest with Be
     sql("DROP table IF EXISTS carbonCahe")
   }
 
-
+  test("Test query with parallel index load") {
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_LOAD_INDEXES_PARALLEL, "true")
+    sql("drop table if exists parallel_index_load")
```

Comment:
Please add drop table to afterALL also  and revert 'CARBON_LOAD_INDEXES_PARALLEL' to default at end of the testcase

Reply:
Done. Have added `afterEach` to ensure all the tables are dropped at the end of testcase. And reverted `CARBON_LOAD_INDEXES_PARALLEL` to default as well